react-frontend/package.json

```diff
@@ -14,6 +14,7 @@
     "react-bootstrap": "^1.0.0",
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.0",
+    "react-router-dom": "^5.1.2",
```

Comment:
Please remove the empty spaces

Reply:
sorry I don't get this which empty spaces?